docs/platform/disk-images/copying-a-disk-image-to-a-different-account/index.md

```diff
@@ -88,56 +127,59 @@ Wait for the transfer to complete. Note that this process can take a while, depe
 
 ## Verifying the Disk
 
-After the file transfer has completed, you should verify the disk by mounting it on the receiving Linode. Here's how:
+After the file transfer has completed, you should verify the disk by mounting it on the *receiving* Linode. Ensure that the [Network Helper](/docs/platform/network-helper/) is on for the *receiving* Linode or the [Static IP](/docs/networking/linux-static-ip-configuration/) is configured to enable networking with the new IP addresses.
+
+1.  Connect to the *receiving* Linode through SSH as `root`.
+
+1. To check that the disks are transferred successfully
```

Comment:
This point might need to be combined with the point below it, as it stands it's just a sentence fragment.